conans/cli/commands/create.py

```diff
@@ -14,7 +15,17 @@
 from conans.util.files import chdir, mkdir
 
 
-@conan_command(group=COMMAND_GROUPS['creator'])
+def json_create(info):
+    deps_graph, profile_build, profile_host = info
+    result = {
+        "profile_build": profile_build.serialize(),
+        "profile_host": profile_host.serialize(),
+        "deps_graph": deps_graph.serialize(),
+    }
+    return json.dumps(result, indent=4)
```

Comment:
can it output a package revision created?

Reply:
The `deps_graph` has that information.

Reply:
thanks. it wasn't obvious, I was thinking it's just an information about dependencies.
is it something like `deps_graph[<package name>].prev`?

Reply:
Yeah, I'm working on that. I only opened the Draft to see all the tests are running ok. Now, let me think of the final format. I hope it will be clearer later.